crates/sui-open-rpc/samples/transactions.json

```diff
@@ -53,103 +53,60 @@
-              "objectId": "0xd77a71ae5148faecb61b22b65432fd84a97e2972",
+              "objectId": "0xe33ff25bdffba609a83a5327ee565af645eb019d",
               "version": 1,
-              "digest": "CWR+Q2LYPSXfQIbn7oTaCG0z9guVfqUTlK5Ohpmvbjc="
+              "digest": "/CXzljbZPnlaZVRO39h89t7gBsOT9/k/zmgbIFpgZG0="
             }
           }
         ],
         "mutated": [
           {
             "owner": {
-              "AddressOwner": "0x51372a6dcf5799e4c39f7cd6bc68e27bdc92728c"
+              "AddressOwner": "0x08177f0d3cb878f43ed6f6d1990629b0ce3d3122"
             },
             "reference": {
-              "objectId": "0x02f7f9837d57a46a77f59b3fe387cad5dc426ab4",
+              "objectId": "0x0bb7abe79e65f28206759931b2cc00e0bab373f4",
               "version": 1,
-              "digest": "+x9cE85GfbUOf/875+FoEtfk6/DHhvtstrMgblewHPg="
+              "digest": "HpZw3djcMnRRXwCk9RGbxoRDDv/UBmpOV4KIpZtEHaM="
             }
           }
         ],
         "gasObject": {
           "owner": {
-            "AddressOwner": "0x51372a6dcf5799e4c39f7cd6bc68e27bdc92728c"
+            "AddressOwner": "0x08177f0d3cb878f43ed6f6d1990629b0ce3d3122"
           },
           "reference": {
-            "objectId": "0x02f7f9837d57a46a77f59b3fe387cad5dc426ab4",
+            "objectId": "0x0bb7abe79e65f28206759931b2cc00e0bab373f4",
             "version": 1,
-            "digest": "+x9cE85GfbUOf/875+FoEtfk6/DHhvtstrMgblewHPg="
+            "digest": "HpZw3djcMnRRXwCk9RGbxoRDDv/UBmpOV4KIpZtEHaM="
           }
         },
         "events": [
           {
-            "type_": "0x2::DevNetNFT::MintNFTEvent",
-            "contents": [
-              215,
-              122,
-              113,
-              174,
-              81,
-              72,
-              250,
-              236,
-              182,
-              27,
-              34,
-              182,
-              84,
-              50,
-              253,
-              132,
-              169,
-              126,
-              41,
-              114,
-              81,
-              55,
-              42,
-              109,
-              207,
-              87,
-              153,
-              228,
-              195,
-              159,
-              124,
-              214,
-              188,
-              104,
-              226,
-              123,
-              220,
-              146,
-              114,
-              140,
-              11,
-              69,
-              120,
-              97,
-              109,
-              112,
-              108,
-              101,
-              32,
-              78,
-              70,
-              84
-            ]
+            "moveEvent": {
+              "type": "0x2::DevNetNFT::MintNFTEvent",
+              "fields": {
+                "creator": "0x08177f0d3cb878f43ed6f6d1990629b0ce3d3122",
+                "name": "Example NFT",
+                "object_id": "0xe33ff25bdffba609a83a5327ee565af645eb019d"
```

Comment:
nit: we should make this camelcase

Reply:
this "object_id" field name is from the move object itself, convert this to camelcase will create a discrepancy between the object and the object definition in move code, it might cause confusions?

Reply:
this "object_id" field name is from the move object itself, convert this to camelcase will create a discrepancy between the object and the object definition in move code, it might cause confusions?